packages/python/BUILD

```diff
@@ -6,7 +6,7 @@
 
 
 def build(ctx):
-    srcdir = ctx.extract('Python-3.6.0a2')
+    srcdir = ctx.extract('Python-3.6.0a3')
```

Comment:
I'd rather leave it as is - _obviously_ hardcoded, and put a more general fix in versions.py for non-integer patch levels.